src/client.ts

```diff
@@ -80,7 +80,7 @@ export class SubscriptionClient {
   private reconnecting: boolean;
   private reconnectionAttempts: number;
   private backoff: any;
-  private connectionCallback: any;
+  private connectionCallback: (error?: Error) => void | undefined;
```

Comment:
```suggestion
  private connectionCallback: (error?: Error) => void;
```

Type `void` is "alias" for `null | undefined`

Reply:
Thanks!